packages/nimble-components/src/table/components/row/index.ts

```diff
@@ -109,6 +110,10 @@ export class TableRow<
     @observable
     public readonly selectionCheckbox?: Checkbox;
 
+    /** @internal */
+    @observable
+    public readonly expandCollapseButton?: HTMLElement;
```

Comment:
I don't see this being referred to by anything. Is this necessary?

Reply:
Looks like I can remove it. Probably a remnant of when the expand/collapse button was tab-focusable.

Reply:
Removed